keras/ops/numpy_test.py

```diff
@@ -4564,10 +4586,10 @@ def test_elementwise_unary_indexed_slices_correctness(
 
     @parameterized.named_parameters(OTHER_UNARY_OPS_TESTS)
     @pytest.mark.skipif(
-        backend.backend() == "tensorflow", reason="Temporary, XLA error"
+        testing.tensorflow_uses_gpu(), reason="Temporary, XLA error"
```

Comment:
@fchollet I reenabled these tests to see the error message, but they're passing right now.